ui/app/pages/swaps/build-quote/build-quote.js

```diff
@@ -215,6 +218,11 @@ export default function BuildQuote({
     );
   };
 
+  const tokenLinkForChain = createTokenTrackerLinkForChain(
```

Comment:
This will only work for the Infura networks. You'll need to use `createCustomTokenTrackerLink` if on a custom network.

Reply:
fixed in 82eb5364b

Reply:
I think you may have misunderstood - I meant that `createCustomTokenTrackerLink` is needed for custom networks, but `createTokenTrackerLinkForChain` is still needed for Infura networks.

Also the wrong parameters are now being passed into `createCustomTokenTrackerLink`

Reply:
Ah, sorry for the mix up. Part of my confusion was that we were still on etherscan-link v1.5.0 and I was looking at the readme  in the node modules directory.

To use `createCustomTokenTrackerLink` we need etherscan-link v2.0.0. I created a new PR to add that #10747, and for now am basing this branch off of that.

And I've added a commit which ensures that the correct links are created for the current chain 60f1d347c

video of it working on mainnet, bsc and testnet is here: https://streamable.com/jkn155

Reply:
Ah, whoops, didn't realize it wasn't updated yet